src/services/EcoBridge/Socket/Socket.utils.ts

```diff
@@ -35,7 +35,8 @@ const OVERRIDES = [
 ]
 
 export const getBestRoute = (routes: Route[], tokenData?: TokenPriceResponseDTO, toTokenDecimals?: number) => {
-  if (routes.length === 1 || !tokenData || !toTokenDecimals) return routes[0]
+  if (routes.length === 1 || !tokenData || !toTokenDecimals || typeof tokenData.result.tokenPrice === 'object')
```

Comment:
Just to be clear, what type should `tokenPrice` be? In this case we are allowing values as `string`, `number` or `undefined`, etc. Maybe we should verify if it's the type we are expecting. This is just a suggestion 😄

Reply:
`tokenPrice` should be a `number` so I think I can change it to `typeof tokenData.result.tokenPrice !== "number"`.
Good catch mate :)

Reply:
fixed [e95336f](https://github.com/levelkdev/swapr-dapp/pull/1352/commits/e95336feef33ff0e90c698b605cc9d077591267e)